_contentTemplates/common/click-events.md

```diff
@@ -63,6 +63,7 @@ Use `OnRowContextMenu` to [integrate the Context menu]({%slug contextmenu-integr
 | Property | Type | Description |
 |---|---|---|
 | `EventArgs` | `EventArgs` | This object maps to `MouseEventArgs` or `KeyboardEventArgs` depending on the user action. |
+| `Field` | `string` | The `Field` property corresponds to the column where the user clicked. If the user clicks on a GridCommandColumn, GridCheckBoxColumn, or columns lacking a `Field` parameter, the `Field` property will be an empty string. |
```

Comment:
- GridCommandColumn -> use either \`GridCommandColumn` or Grid Command Column
- GridCheckBoxColumn -> use either \`GridCheckBoxColumn` or Grid CheckBox Column

Reply:
Consider something simpler like "The `Field` parameter of the clicked column."